src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/02 Advanced Examples.md

```diff
@@ -3,11 +3,22 @@ title: 'Advanced Examples'
 excerpt: 'Advanced Examples using the k6 Scenario API - Using multiple scenarios, different environment variables and tags per scenario.'
 ---
 
-## Using multiple scenarios
+You can use multiple scenarios in one script, and these scenarios can be run in sequence or in parallel.
+So besides helping you model workloads, scenarios can also help you make more flexible and composable test logic.
+Some ways that you can use scenarios include the following:
+- Sequence workloads
```

Comment:
The problem with this (and somewhat with "flexible and composable test logic" above) is that it's not actually that easy to run scenarios "in sequence" :disappointed: we don't have a `startAfter` logic for scenarios (and we probably won't ever have it), just `startTime`... 

Related k6 issue: https://github.com/grafana/k6/issues/1342